### PR TITLE
Add audicy to apps

### PR DESCRIPTION
**Pull request by Jerick120**

### CODE REVIEW

**Comment by stevencrader** [comment]
What podcasting 2.0 features or podcast namespace are being used?

**Comment by ericpp** [comment]
@Jerick120 The apps listed on the Podcast Index page should incorporate features from the Podcasting spec here: https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md or from the supported element list here: https://github.com/Podcastindex-org/web-ui/blob/master/server/data/apps-schema.json#L101-L128

Does your app incorporate some of these features? A Discord bot with Boostagram (Value Tag) or Live Item support would be excellent!

File: server/data/apps.json [hunk 1]
```diff
@@ -1,4 +1,21 @@
 [
+  {
+    "appName": "Audicy",
+    "appType": [
+      "Discord Audio Streaming Bot"
+    ],
```
Comment (ericpp) [inline server/data/apps.json:6]: The app types have to match up with the app types listed in: https://github.com/Podcastindex-org/web-ui/blob/45cce47335def6226640f539e5ac3a2b45816a2d/server/data/apps-schema.json#L24-L45

```suggestion
    "appType": [
      "app",
      "podcast player"
    ],
```

I wonder if a `chat bot` type needs to be added to `apps-schema.json`

File: server/data/apps.json [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+    "appUrl": "https://audicy.xyz",
+    "appIconUrl": "audicylogo.png",
+    "platforms": [
+      "Discord"
+    ],
+    "supportedElements": [
+      {
+        "elementName": "Discord Server",
+        "elementURL": "https://discord.gg/4Vxsf4Y8KK"
```
Comment (ericpp) [inline server/data/apps.json:15]: I would recommend `Search` here along with a link to a page/blog that says it's being used:

```suggestion
        "elementName": "Search",
        "elementURL": "https://audicy.xyz/about"
```

File: server/data/apps.json [hunk 3]
```diff
@@ -1,4 +1,21 @@
 [
+  {
+    "appName": "Audicy",
+    "appType": [
+      "Discord Audio Streaming Bot"
+    ],
+    "appUrl": "https://audicy.xyz",
+    "appIconUrl": "audicylogo.png",
+    "platforms": [
+      "Discord"
```
Comment (ericpp) [inline server/data/apps.json:10]: I think `Discord` would need to be added to the list of platforms here: https://github.com/Podcastindex-org/web-ui/blob/45cce47335def6226640f539e5ac3a2b45816a2d/server/data/apps-schema.json#L55-L81

File: web-ui
Comment (ericpp) [inline web-ui:1]: Was this accidentally removed?

**Review by ericpp — COMMENTED** [review]
Here's some recommendations that fit into apps, platforms, and elements defined in: https://github.com/Podcastindex-org/web-ui/blob/master/server/data/apps-schema.json

**Comment by ericpp** [comment]
> @ericpp Attempted to fix most of the issues.
> 
> > Was this accidentally removed?
> 
> I am getting an error along with the commit, "web-ui submodule does not have a commit checked out" and it ignores the submodule completely. Wasen't able to fix this. I checked for git initializations in the folder but there aren't any.

I'm not really sure what the purpose of that `web-ui` submodule is. It looks like it might've been accidentally added by another app developer in 253dd9c021039c6b77465364e5348c60aca73b92

The rest looks good to me, but I have no real power here. @daveajones will have to take a look at it and decide whether to merge it in.

**Comment by daveajones** [comment]
I’ll check it out today. Thank you for triage @ericpp! 🙏

**Pull request merged**